ReactiveObjC/RACSourceSymbolExtractor.m

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2021 Lightricks. All rights reserved.
+// Created by Ofir Gluzman.
+
+#if DEBUG
+
+#import "RACSourceSymbolExtractor.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+NSString *RACExtractSourceSymbol(NSArray<NSString *> *callStackSymbols) {
```

Comment:
A different approach to do this would be to get the address range of RAC's image in memory and see if the call stack addresses are in the range of this image.

Reply:
Not sure how to get the range of the image.
I see I have this method `_dyld_get_image_header_containing_address` that returns `mach_header *` which I understand should be at the start of the image block in memory - so the address of the `mach_header *` is also the range start ?
Even so, I'm not sure how to get the range end / the size of the image.

Regardless, the `dladdr` implementation is already much faster so not sure if this is needed.

But it will be interesting to know how should this be done :)

Reply:
Answered privately to @ofirgluzman - if anyone else is interested please ask him.